src/Psecio/Parse/Tests/TestLogicalOperatorsFound.php

```diff
@@ -5,35 +5,25 @@
 use Psecio\Parse\TestInterface;
 use PhpParser\Node;
 use Psecio\Parse\File;
+use PhpParser\Node\Expr\BinaryOp\LogicalAnd;
+use PhpParser\Node\Expr\BinaryOp\LogicalOr;
 
 /**
- * Logical operators should be avoided
+ * The logical operators OR and AND should be avoided as they have lower precedence than || and &&
  */
 class TestLogicalOperatorsFound implements TestInterface
 {
     use Helper\NameTrait;
 
-    private static $operators = ['and', 'or', 'xor'];
-
     public function getDescription()
     {
-        return 'Avoid the use of logical operations (XOR, OR, etc) in favor of operators like && and ||';
+        return "Avoid the use of OR and AND in favor of || and && as they may cause subtle bugs due to precedence";
     }
 
     public function evaluate(Node $node, File $file)
     {
-        if ($node instanceof \PhpParser\Node\Expr\BinaryOp\NotIdentical) {
-            // See what's on the line
-            $attr = $node->getAttributes();
-            $lines = $file->getLines($attr['startLine']);
-
-            foreach ($lines as $line) {
-                foreach (self::$operators as $operator) {
-                    if (stristr($line, $operator) !== false) {
-                        return false;
-                    }
-                }
-            }
+        if ($node instanceof LogicalAnd || $node instanceof LogicalOr) {
```

Comment:
Yes it is. But xor has no equivalent to || or &&. I am not sure what best practice is here. Do you have a link to a resource discussing this?